src/sentry/api/endpoints/project_codeowners.py

```diff
@@ -24,6 +24,8 @@
 
 logger = logging.getLogger(__name__)
 
+MAX_RAW_LENGTH = 100000
```

Comment:
```suggestion
# Max length of the codeowners data in bytes
MAX_RAW_LENGTH = 100_000
```

Reply:
hmm actually it's length of characters

Reply:
I'm wondering if maybe we should check bytes here

Reply:
@evanpurkhiser hmm its not a bytestring tho. It'll always be a text string.

Reply:
yeah but it could be if you `encode('utf-8')`. I suppose text length is fine, probably makes more sense.

In that case let's change the comment 'Max string length of the codeowners input'